package.json

```diff
@@ -4,10 +4,10 @@
   "repository": "nearform/node-clinic-bubbleprof",
   "version": "1.8.0",
   "scripts": {
-    "test": "standard | snazzy && tap --no-cov test/*.test.js",
+    "test": "standard | snazzy && tap --no-cov --timeout=50 test/*.test.js",
     "ci-lint": "standard | snazzy",
-    "ci-test": "tap test/*.test.js",
-    "ci-cov": "tap --statements=98 test/*.test.js",
+    "ci-test": "tap --timeout=50 test/*.test.js",
+    "ci-cov": "tap --statements=98 --timeout=60 test/*.test.js",
```

Comment:
why statements?

Reply:
That's just how it was before. 

I assume it's because we wanted coverage at ~98% and felt statements gives more control e.g. multiple statements per line